src/components/Discussions/ReplyCard.tsx

```diff
@@ -113,7 +113,7 @@ export const ReplyCard = (props: IProps & IRecordProps) => {
         onOk={handleSaveEdit}
         onCancel={handleCancelEdit}
         setCanSaveRecording={setCanSaveRecording}
-        fileName={fileName}
+        fileName={fileName(discussion.attributes.subject, discussion.id)}
```

Comment:
This logic may not change when discussion id changes and subject changes since the address of fileName function doesn't change or it will update too frequently. We usually try to have a useMemo value that calculates the fileName.